catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventHandlerDelegate.java

```diff
@@ -137,19 +147,12 @@ public Metacard read(InputStream inputStream) throws CatalogTransformerException
         for (SaxEventHandler eventHandler : eventHandlers) {
             List<Attribute> attributes = eventHandler.getAttributes();
             for (Attribute attribute : attributes) {
-                Attribute tmpAttr;
 
                 /*
-                 * If metacard already has values in the attribute, put them together into a multivalued list,
+                 * If metacard already has values in the attribute, skip the attribute,
                  * instead of simply overwriting the existing values.
                  */
-                if ((tmpAttr = metacard.getAttribute(attribute.getName())) != null) {
```

Comment:
I added a Utility class that will combine common attributes for a handler. The Delegate used to combine all common attributes across all handlers. Since this functionality was removed, common attributes will be combined within the handlers themselves. 
The Delegate will take the first attribute's value or list of values.